.github/workflows/ci.yml

```diff
@@ -82,19 +86,29 @@ jobs:
   pe-parse-windows:
     strategy:
       matrix:
+        build-arch: ["x64", "Win32"]
         build-type: ["Debug", "Release"]
+        build-shared: ["0", "1"]
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
     - name: build
       run: |
         mkdir build
         cd build
-        cmake -G "Visual Studio 16 2019" -A x64 ..
+        cmake `
+          -G "Visual Studio 16 2019" `
+          -A ${{ matrix.build-arch }} `
+          -DBUILD_SHARED_LIBS=${{ matrix.build-shared }} `
+          ..
```

Comment:
Never knew the ` (back-tick) character was used for multi-line commands in Windows 👀

Reply:
Yeah, I learned that as part of the uthenticode CI setup 😅

Reply:
(It's a powershell specific thing, apparently).